### ISSUE TITLE
The command palette blocks the window

### ISSUE THREAD

**Issue by nblackburn** [fman-community/fman#125]
The command palette blocks the window preventing it from being moved etc...

> All other implementations of command palettes (Sublime, Atom, VS Code etc..) do not do this.

**Comment by mherrmann** [comment]
This is somewhat related to #115.

**Comment by nblackburn** [comment]
Any progress on this, it still happens on Windows?

**Comment by mherrmann** [comment]
No new progress yet, I'm afraid. The current implementation makes the Command Palette a modal dialog. So it behaves like a "are you sure you want to quit?" dialog. A new implementation - probably the goal of this issue - would be to draw the Command Palette inside fman's main window, over the rest of the GUI. For now, you'll have to close the Command Palette with <kbd>Escape</kbd> if you want to move/resize the window.

**Comment by nblackburn** [comment]
@mherrmann Yeah, that sounds like what is required, i will leave this open for now.

**Comment by mherrmann** [comment]
Yes, sure. You're [not the only person](https://www.macworld.co.uk/download/system-desktop-tools/fman-093-3331358/) who finds this a pain.

**Comment by briankb** [comment]
Thought I needed to add a new issue but found this one via the Related Issues search. 

Usually when you click outside the area of a child window it is closed and that's what I expected with fman but kept hearing the Exclamation sound (very annoying). You mentioned already that is was because the code considers this a confirmation window which explains the behavior. 

It really does interfere with the nice "flow" that you get with fman and all the keyboard shortcuts to have have to remember to hit escape. And that sound! it's very distracting and takes you out of whatever zone you were in. 

I hope you will consider fixing this sooner than later. I know Windows users aren't always the priority for Mac devs! 👅

Is there another dialog or dialog setting that could "escape" it without a lot of code refactoring?

**Comment by mherrmann** [comment]
I can see how it's annoying. But you can just press <kbd>Esc</kbd> to close the command palette instead of clicking outside with the mouse. You shouldn't be using the mouse anyways ;)

And I'm sorry no, there's no setting.
